presto-hive/src/test/java/com/facebook/presto/hive/benchmark/FileFormat.java

```diff
@@ -30,10 +30,10 @@
 import com.facebook.presto.hive.HiveStorageFormat;
 import com.facebook.presto.hive.HiveType;
 import com.facebook.presto.hive.HiveTypeName;
-import com.facebook.presto.hive.HiveTypeTranslator;
 import com.facebook.presto.hive.RecordFileWriter;
 import com.facebook.presto.hive.TypeTranslator;
 import com.facebook.presto.hive.benchmark.HiveFileFormatBenchmark.TestData;
+import com.facebook.presto.hive.metastore.HiveTypeTranslator;
```

Comment:
There're many files whose change was just the HiveTypeTranslator move. Is the move necessary? If not, can we keep that class at its original location and remove changes in all files caused by the move?

Reply:
Moved it back and implemented a simple translateToHiveType in PartitionFilterBuilder instead